src/main/kotlin/org/jetbrains/research/ml/dataset/anonymizer/util/CodetrackerFileUtil.kt

```diff
@@ -4,4 +4,8 @@ object CodetrackerFileUtil {
     enum class Column(val key: String) {
         FRAGMENT("fragment")
     }
+
```

Comment:
Do we need an object for that? Why? We can use these enum classes in a more pure way without adding CodetRackerFileUtil in the beginning